src/Director/BodyTo/BodyToAffiliateSeller.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PlugAndPay\Sdk\Director\BodyTo;
+
+use PlugAndPay\Sdk\Contract\BuildMultipleObjectsInterface;
+use PlugAndPay\Sdk\Contract\BuildObjectInterface;
+use PlugAndPay\Sdk\Entity\AffiliateSeller;
+use PlugAndPay\Sdk\Entity\AffiliateSellerInternal;
+use PlugAndPay\Sdk\Enum\SellerStatus;
+use PlugAndPay\Sdk\Traits\BuildMultipleObjects;
+
+class BodyToAffiliateSeller implements BuildObjectInterface, BuildMultipleObjectsInterface
+{
+    use BuildMultipleObjects;
+
+    public static function build(array $data): AffiliateSeller
+    {
+        $seller = (new AffiliateSellerInternal(false))
+            ->setId($data['id'])
+            ->setName($data['name'])
+            ->setEmail($data['email'])
+            ->setDeclineReason($data['decline_reason'] ?? null)
+            ->setProfileId($data['profile_id'])
+            ->setStatus(SellerStatus::from($data['status']))
+            ->setPayoutMethods($data['payout_methods'] ?? []);
```

Comment:
ik denk dat deze nog los moet met een isset, dit wordt niet returned in affiliates/sellers.

```
{
            "id": 5,
            "decline_reason": "Consequatur non rerum in cupiditate voluptas molestiae fuga.",
            "email": "vdeswart@vannortheim.org",
            "name": "Samuel Said",
            "profile_id": 13,
            "status": "accepted"
        }
```